BoneBreakerMergeShop[MEM].cs

Hassan0Kamel opened the pull request:
it does farm all the items in mergeshop bonebreaker, 1051 (require membership ofc), may need to be added to membership dailies kit btw

Hassan0Kamel committed:
BoneBreakerMergeShop[MEM].cs

it does farm all the items in mergeshop bonebreaker, 1051 (require membership ofc)

Hassan0Kamel committed:
Update BoneBreakerMergeShop[MEM].cs

edited tag Membership to member + added tags daily and dailies

wtffidy commented on Other/MergeShops/BoneBreakerMergeShop[MEM].cs:
```diff
@@ -0,0 +1,95 @@
+                    {
+                        Core.EnsureAccept(3898);
+                        Core.HuntMonster("bonebreaker", "Undead Berserker", "Warrior Defeated", 5);
+                        CorHeader.EnsureComplete(3898);
```
Core >.>

Hassan0Kamel replied:
oh the auto choosing function when I press .  >.<

wtffidy commented on Other/MergeShops/BoneBreakerMergeShop[MEM].cs:
```diff
@@ -0,0 +1,95 @@
+using System.Runtime.ExceptionServices;
```
extra generated usings need deleted

wtffidy commented on Other/MergeShops/BoneBreakerMergeShop[MEM].cs:
```diff
@@ -0,0 +1,95 @@
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv = new();
```
the include for dailies was added but not the private line here which is required

Hassan0Kamel committed:
Update BoneBreakerMergeShop[MEM].cs

1- deleted the magically generated things at the top
2- added the private class of coredailies
3- edited the core ensure complete because it did choose another core function automatically

wtffidy commented on Other/MergeShops/BoneBreakerMergeShop[MEM].cs:
```diff
@@ -0,0 +1,92 @@
+                    {
+                        Core.EnsureAccept(3898);
+                        Core.HuntMonster("bonebreaker", "Undead Berserker", "Warrior Defeated", 5);
+                        Cor.EnsureComplete(3898);
```
Core? core..? Core! core 🔴

wtffidy replied:
Core

Hassan0Kamel committed:
Update BoneBreakerMergeShop[MEM].cs

added the missing e in Cor(e).EnsureComplete
